packages/near-membrane-base/src/environment.ts

```diff
@@ -128,6 +159,18 @@ export class VirtualEnvironment {
         }
     }
 
+    link(path: string) {
+        const keys = path.split('.');
```

Comment:
this is not super pretty, but it is only used during initialization.

yes yes, I know... `.split` should be a cached method with `ReflectApply`, will do that soon @jdalton jajajaja

Reply:
another alternative here is to provide an array of property keys, so symbols can also work fine.

Reply:
I had this with Lodash and accepting a path. Passing an array is the most flexible and less-worky way to do it. So an array path would be nice. We (Rick and I can follow-up with that kinda jazz)